src/Meshes.jl

```diff
@@ -78,6 +79,8 @@ export
   # connectivities
   Connectivity,
   polytopetype, connect, materialize,
+  OrderingConvention, GMSH,
+  connectivity,
```

Comment:
I should probably separate that from the connectivity exports since it is related to conventions, right?

Reply:
Yes, a section for ordering conventions, and any function related to that if necessary.